### ISSUE TITLE
Issue with non-integer partial timings

### ISSUE THREAD

**Issue by giovannipizzi** [QEF/qeschemas#1]
I have a QE 6.6 calculation where the schema issues this warning:
```
 | failed validating 0 with <function positive_int_validator at 0x7fba9e098950>:
 | 
 | Reason: value must be positive.
 | 
 | Instance:
 | 
 |   <partial calls="0" label="electrons">
 |     <cpu>1.390724593400000e4</cpu>
 |     <wall>1.395315800213814e4</wall>
 |   </partial>
 | 
 | Path: /{http://www.quantum-espresso.org/ns/qes/qes-1.0}espresso/timing_info/partial
```
Indeed, the partial times are not integers, see e.g. the corresponding text output:
```
     electrons    :  13907.25s CPU  13953.16s WALL (       1 calls)
```

**Comment by giovannipizzi** [comment]
By the way, shouldn't the `calls="0"` be 1 instead, according to the text output?

**Comment by pietrodelugas** [comment]
you are right  that's why we are getting the issue.  Probably the  counter is wrong by one.

**Comment by pietrodelugas** [comment]
Giovanni
 could you send me the xml file ? I am not able to reproduce the 0 calls in my test cases

**Comment by giovannipizzi** [comment]
Sorry @pietrodelugas I had missed some notifications

here are ins and outs
[aiida.in.txt](https://github.com/QEF/qeschemas/files/5524300/aiida.in.txt)
[aiida.out.txt](https://github.com/QEF/qeschemas/files/5524301/aiida.out.txt)
[data-file-schema.xml.txt](https://github.com/QEF/qeschemas/files/5524302/data-file-schema.xml.txt)
